src/components/Button.vue

```diff
@@ -0,0 +1,21 @@
+<template>
```

Comment:
I think you were trying to encapsulate the button styling, but this is not a good way of achieving this, because...

- You are relying on events and attributes you can pass directly to HTML's `<button>`, this is fragile and would require the parent to change if you change the implementation of this component.
- This can be easily achieved by having this styling on `mani.scss` or a similar file (outside components)